shared/tasks/styles.mjs

```diff
@@ -78,6 +78,7 @@ export async function compileStylesheet ([modulePath, { srcPath, destPath, fileP
 
       // Resolve @imports via
       loadPaths: [
+        paths.src,
```

Comment:
We'll swap this with `node_modules/govuk-frontend` once we've changed our package structure